xrpl/models/utils.py

```diff
@@ -1,9 +1,44 @@
 """Helper util functions for the models module."""
+
+import re
 from dataclasses import dataclass, is_dataclass
-from typing import Any, Dict, List, Type, TypeVar, cast
+from typing import Any, Dict, List, Optional, Pattern, Type, TypeVar, cast
+
+from typing_extensions import Final
 
 from xrpl.models.exceptions import XRPLModelException
 
+# Note: This regex is used in DIDSet and Credential-related transactions. In addition to
+# this regex, Credential-related transactions requires the input strings to have
+# positive (non-zero) length
```

Comment:
This comment isn't very future proof. I would rewrite it to be more general, so we don't have to remember to update the comment if this regex is used elsewhere.

Reply:
Okay, what do you recommend? Should I remove references to specific transactions like `DIDSet`?
Are you thinking of cases where `DIDSet` transaction undergoes updates? In that case, I can remove this comment, its not required tbh

Reply:
addressed in [8e496e1](https://github.com/XRPLF/xrpl-py/pull/759/commits/8e496e18ddddb58ac27457210e10716cdae80eeb)